templates/csharp/api.mustache

```diff
@@ -12,22 +12,50 @@ using Adyen.Service.Resource;
 
 namespace {{packageName}}.{{apiPackage}}
 {
+{{#operations}}
+    /// <summary>
+    /// {{classname}} Interface
+    /// </summary>
+    public interface I{{classname}}
+    {
+        {{#operation}}
+{{>method_documentation}}
+        {{#returnType}}
+        /// <returns><see cref="{{{returnType}}}"/>.</returns>
+        {{/returnType}}
+        {{#isDeprecated}}
+        [Obsolete]
+        {{/isDeprecated}}
+        {{{returnType}}}{{^returnType}}void{{/returnType}} {{#lambda.pascalcase}}{{vendorExtensions.x-methodName}}{{/lambda.pascalcase}}({{>api_parameters}});
+        
+        {{#supportsAsync}}
+{{>method_documentation}}
+        /// <param name="cancellationToken"> A CancellationToken enables cooperative cancellation between threads, thread pool work items, or Task objects.</param>{{#returnType}}
```

Comment:
Is this hard coded ?

Reply:
Correct, we should add the cancellationToken to our async methods on service-level and client-level. We have a ticket open to make that change as well 👍

Reply:
Cool thanks guys @Kwok-he-Chu @jillingk !